LondonBritishLibrary/orient/BLorient11718A.xml

```diff
@@ -0,0 +1,226 @@
+                                        የሚያስተምሩት፡ ምንድርነው። </incipit>
+                                    <textLang xml:lang="am"/>
+                                </msItem>
+                                <msItem xml:id="p1_i1.2">
+                                    <locus from="9" to="82"/>
+                                    <title>The sixty questions and answers</title>
+                                    <incipit xml:lang="am">፩ጥያቄ። በናንተ፡ ቤተ፡ ክርስቲያናት፡ ለሕፃናትና፡ ላልተማሩ፡ <gap reason="ellipsis" unit="chars" quantity="19"/>
+                                        ምላሽ። በኛ፡ ቤተ፡ ክርስቲያን፡ ለሕፃናትና፡ ላልተማሩ፡ ሰዎች፡ መጀምሯ፡ ስለ፡ እግዚአብሔር፡ ባሕሪ፡ የሚያስተምሩት፡ እንዴህ፡ ነው። እግዚአብሔር፡ በግብሩ፡ 
+                                        በስራው፡ በመንፈስ፡ ይገለጥልናልና፡ በዓይነ፡ ሃይማኖት፡ እናየዋለን፡ በዓይነ፡ ልብና፡ እናውቀዋለን፡ እንጂ፡ እግዚአብሔርን፡ በባሕረው፡ ለማወቅ፡ እንኳን፡ ሥጋ፡
+                                        ለለበሰ፡ ሰው። ለቅዱሳን፡ መላእክትም፡ አይቻላቸውም። ዕብ፡ ፲፩ምዕ፡ ፩፡ ፫፡ ቍ። ኢዮ፡ ፲፩ምዕ፡ ፯ቍ። ኢዮ። ፴፯ምዕ፡ ፭፡ ፳፫ቍ። ምዝ፡ ፸፯ቍ። ፲፱። መዝ፡ ፻፵፭፡ 
+                                        ቍ፡ ፫። ኢሳ፡ ፵፡ ምዕ፡ ፳፰ቍ። ማቴ፡ ፲፩ምዕ፡ ፳፯ቍ። ሮሜ፡ ፲፩ምዕ፡ ፴፫ቍጥ፤ </incipit>
+                                    <textLang xml:lang="am"/>
+                                </msItem>
+                            </msItem>
+                        </msContents>
+                        <physDesc>
+                            <objectDesc form="Codex">
+                                <supportDesc>
+                                    <support>
+                                        <material key="paper"/>
+                                    </support>
+                                    <extent>
+                                        <measure unit="page">82+1+8</measure>    
+                                    </extent>
+                                    <foliation>Unlike the other parts, the first codicological unit is paginated beginning from page 1 to 82, while the back of 
+                                        page 1 is not numbered, leaving seven unnumbered pages at the end.</foliation>
+                                </supportDesc>    
+                            </objectDesc>
+                            <handDesc>
+                                <handNote script="Ethiopic" xml:id="h1">
+                                    <locus from="2" to="82"/>
+                                    <desc>Large careful handwriting.</desc>
+                                    <date notBefore="1923" notAfter="1924"/>
+                                </handNote>
+                            </handDesc>
+                            <additions>
+                                <list>
+                                    <item xml:id="a1">
+                                        <locus target="#1"/>
+                                        <desc>A pencil note by <persName ref="PRS5226haylase"><roleName type="title">rās</roleName> 
+                                            Tafari Makʷannǝn</persName>: 'Answers given by Ethiopian Doctors to the questions addressed by  
```

Comment:
I would shape this note as other a-additional notes. Attribute `unclear` can be used, if it does not fit the system of the genres; is "Answers..." you translation or continuation of the note? (if so, I would not split it).

Reply:
I think, it is letter (a1) and comment (a2, previously e1).